RdkServicesTest/Scripts/build.sh

```diff
@@ -109,12 +121,21 @@ buildAndInstallRdkservices() {
     -DCMAKE_INSTALL_PREFIX="${THUNDER_INSTALL_DIR}/usr" \
     -DCMAKE_MODULE_PATH="${THUNDER_INSTALL_DIR}/tools/cmake" \
     -DCOMCAST_CONFIG=OFF \
+    -DPLUGIN_DEVICEDIAGNOSTICS=ON \
     -DPLUGIN_LOCATIONSYNC=ON -DPLUGIN_LOCATIONSYNC_URI="http://jsonip.metrological.com/?maf=true" \
     -DPLUGIN_PERSISTENTSTORE=ON \
     -DPLUGIN_SECURITYAGENT=ON \
-    -DRDK_SERVICES_TEST=ON
+    -DRDK_SERVICES_TEST=ON \
+    -DCMAKE_BUILD_TYPE=$MODE
+
+  make -C build/rdkservices $THREADS && make -C build/rdkservices install $THREADS
+}
 
-  make -C build/rdkservices && make -C build/rdkservices install
+installMockServer() {
+  cd "$ROOT"
+  docker pull mockserver/mockserver
```

Comment:
this adds too many new dependencies like node, npm, docker. Since we already rely on python, can we use it instead?

Reply:
@mateusz-adamski-red we need something very basic, like https://stackoverflow.com/a/47232498. Or just mock libcurl.